scripts/ci/run-vagrant-tests.sh

```diff
@@ -23,8 +23,20 @@ trap vagrant_cleanup EXIT
 export ANALYZER_COUNT=2
 export AGENT_COUNT=1
 
-for mode in dev binary package container
+for mode in binary package container
 do
+  if [ "$DEVMODE" == "true" ]; then
+    case $mode in
+      dev)
```

Comment:
dev is not in the list, I guess it should be binary

Reply:
fixed